src/handbook/sales/index.md

```diff
@@ -56,4 +56,10 @@ For each sales opportunity a clear answer for the customers needs and wants shou
 If, and only if, the answers are known and added as notes to HubSpot, an
 opportunity is added to the HubSpot deals board.
 
+## HubSpot Properties
+
+`Outbound` is a custom property that's set to `Yes` when the first meeting with the contact came through outbound
+drip campaigns or other outbound lead-gen actions. This property will be set to `Yes` when the contact was in HubSpot
+through other marketing activities too, but wasn't nurtured to the point of a meeting yet.
```

Comment:
What does this mean? If the contact was in Hubspot already isn't that the definition of Inbound?

Reply:
I see what you mean. The property sources on creation are still set by HS. The intent is to understand which meetings are SDR generated. I bet we can rename the property. Any suggestions @iskerrett ?

Reply:
Typically Inbound leads do not have a contact owner assigned. If an SDR starts working a contact, they should assign themselves as the owner. I don't think we need a new property for this?

Reply:
@zackwasli Can we rename the field to "First Meeting Outbound".

Reply:
Actually, let's go with "Activation Outbound"